projects/canopy/src/lib/carousel/carousel.component.scss

```diff
@@ -0,0 +1,66 @@
+.lg-carousel {
+  overflow: hidden;
+
+  &__wrapper {
+    position: relative;
+    display: flex;
+    align-items: stretch;
+  }
+
+  &__navigation {
+    padding-top: var(--space-sm);
+    display: flex;
+    justify-content: center;
+  }
+
+  &__centered {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__button {
+    padding: 0;
+    border: none;
+    background-color: transparent;
+    margin: 0;
+    width: var(--space-xxl);
+    height: var(--space-xxl);
+    cursor: pointer;
+
+    &:focus-visible {
+      outline: none;
+    }
+
+    &__disk {
+      width: var(--space-xxl);
+      height: var(--space-xxl);
+      border-radius: 100%;
+    }
+
+    &:hover:not([disabled]) {
+      .lg-carousel__button__disk {
```

Comment:
not something that particularly bothers me but I know we try to avoid more than 3 levels of nesting in our sass

Reply:
no problem, that's easily changed